### ISSUE TITLE
Feature Suggestion: Provide an option to provide a maximum time limit for unfreezing?

### ISSUE THREAD

**Issue by ndattani** [tripleee/sloshy#32]
I wrote "feature suggestion" instead of "feature request" because I'm currently a bit indifferent as to whether or not I want to "request" for this to be implemented, but [I came across a user] who might like for sloshy to post a maximum of three times in a row](https://chat.stackexchange.com/transcript/message/62274319#62274319) and then to let the room freeze if no one writes any messages after those 36 days of inactivity.

**Comment by tripleee** [comment]
I doubt this is worth automating. If you have chat rooms which should not be kept alive any longer, just remove them from Sloshy's list.

It would not technically be hard to add an end date to the per-room configuration data, so I'm leaving this open for a while, and welcome PRs if somebody should genuinely want to spend time on this.

**Comment by ndattani** [comment]
Yes, what I had in mind was adding an "end date" for cases like the above one, where the user only wants the room open for 36 days (for example). For election rooms. one might want the room open until the election is finished, but not necessarily much longer after that.
